articles/postgresql/howto-restore-dropped-server.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Restore a dropped Azure Database for PostgreSQL server
+description: This article describes how to restore a dropped server in Azure Database for PostgreSQL using the Azure portal.
+author: Bashar-MSFT
+ms.author: bahusse
+ms.service: postgresql
+ms.topic: how-to
+ms.date: 11/03/2020
+---
+# Restore a dropped Azure Database for PostgreSQL server
+
+When a server is dropped, the database server backup can be retained up to five days in the service. The database backup can be accessed and restored only from the Azure subscription where the server originally resided. The following recommended steps can be followed to recover a dropped PostgreSQL server resource within 5 days from the time of server deletion. The recommended steps will work only if the backup for the server is still available and not deleted from the system. 
```

Comment:
I think the term dropped is for a database using SQL commands, but in our nomenclature, a server is **deleted**. This impacts the title, description, H1 heading, and several other paragraphs in the text.